lib/qa/authorities/linked_data/find_term.rb

```diff
@@ -43,34 +42,36 @@ def initialize(term_config)
       def find(id, language: nil, replacements: {}, subauth: nil, jsonld: false)
         raise Qa::InvalidLinkedDataAuthority, "Unable to initialize linked data term sub-authority #{subauth}" unless subauth.nil? || term_subauthority?(subauth)
         @language = language_service.preferred_language(user_language: language, authority_language: term_config.term_language)
-        url = authority_service.build_url(action_config: term_config, action: :term, action_request: normalize_id(id), substitutions: replacements, subauthority: subauth, language: @language)
+        @id = id
+        url = authority_service.build_url(action_config: term_config, action: :term, action_request: normalize_id, substitutions: replacements, subauthority: subauth, language: @language)
         Rails.logger.info "QA Linked Data term url: #{url}"
         load_graph(url: url)
         return "{}" unless full_graph.size.positive?
         return full_graph.dump(:jsonld, standard_prefixes: true) if jsonld
-        parse_term_authority_response(id)
+        filter_graph
```

Comment:
Filtering the graph was happening at the same time the full graph was read in.  If the full graph is being returned, there is no need to perform the filtering, so moved it to a separate method that is only called after it is determined we aren't going to pass back the full graph.